File: zad1_python/exporter.py
__author__ = 'Makda'

class Exporter:
    data = ""

    def __init__(self, data):
        self.data = data

    def export_data(self):
        a = self.data
        self.data = ""
        return a

class TextExporter(Exporter):
    def __init__(self, data):
        Exporter.__init__(self, data)

File: zad1_java/src/com/magda/Exporter.java
package com.magda;

/**
 * Created by magdgar on 2016-01-19.
 */
public abstract class Exporter {
    protected Data exportedData;

    public Exporter(Data exportedData){
        this.exportedData = exportedData;
    }

    public abstract Data exportData();
}
